tests/storage_test.go

```diff
@@ -142,8 +133,9 @@ var _ = Describe("Storage", func() {
 				RunVMIAndExpectLaunch(vmi, false, 90)
 
 				expecter, err := tests.LoggedInCirrosExpecter(vmi)
-				Expect(err).To(BeNil())
+				Expect(expecter).NotTo(BeNil())
```

Comment:
We should not get an error AND an expecter? That is counterintuitive in the go world. Let's always close the expecter in `LoggedInCirrosExpecter` if such an error occurs where we already have the expecter.